compiler/rustc_hir_typeck/src/fn_ctxt/mod.rs

```diff
@@ -49,6 +49,8 @@ pub struct FnCtxt<'a, 'tcx> {
     /// eventually).
     pub(super) param_env: ty::ParamEnv<'tcx>,
 
+    pub(super) parent_module: DefId,
```

Comment:
please add a comment stating that this is used for visibility information in diagnostics or sth.